src/modules/ColorTemperature/Module.js

```diff
@@ -38,17 +38,12 @@ module.exports = function ColorTemperature(options, UI) {
             for (let i = 0; i < pixels.shape[0]; i++) {
                 for (let j = 0; j < pixels.shape[1]; j++) {
 
-                    r_data = pixels.get(i, j, 0)
-                    r_new_data = (255 / r) * r_data
-                    pixels.set(i, j, 0, r_new_data)
+                    var rgbdata = [pixels.get(i, j, 0),pixels.get(i, j, 1),pixels.get(i, j, 2)];
+                    rgbdata[0] = (255 / r) * rgbdata[0];
```

Comment:
You can use `rgbdata[0] *= (255 / r)` or `rgbdata.forEach(data => data * (255 / r)`.